static/script/devices/mediaplayer/seekfinishedemitevent.js

```diff
@@ -37,7 +37,7 @@ define(
                 } else {
                     this._timeoutHappened = true;
                 }
-                oldSetSource.call(this, mediaType, url, mimeType, opts);
+                oldSetSource.call(this, mediaType, url, mimeType, videoContainer, opts);
```

Comment:
This is a major version change (API breaking), could this not go as an optional last param?

Reply:
@subsidel this is intentional - we don't want it to be optional, as this will help decouple the TAL Media Player from the TAL Application object sooner rather than later. 

It's also trivial to adopt this change if you want to retain the same behaviour (pass in `RuntimeContext.getCurrentApplication().getRootWidget().outputElement`).

Finally, there are no other changes going out with this release, so dependent projects can stick with `8.2.1` without missing out on anything.

Reply:
It also feels odd that this is being done on `setSource` rather than in `init`, where to render the element isn't part of the source of the media file, why is it in there?

Reply:
TAL Media Player acts as a singleton - `Device.getMediaPlayer()` always returns the same instance, so passing it into the init would be difficult (as TAL calls `init`, not the consumer of TAL) and would limit video to be rendered in 1 place for an entire session.

Reply:
Perhaps a rename then?
eg `MediaPlayer.renderSourceInto(...sourceArgs, DOMElement)`

Reply:
Looking at everything that function does, ```setSource``` doesn't feel right, neither does ```renderSourceInto```, how about ```initialiseMedia``` or simmilar.

Reply:
Yep, I like the rename to ```initialiseMedia```. Will get that in now.